### DISCUSSION TITLE
Author Roadmap

### DISCUSSION THREAD

**Discussion by maxpearl** [bookwyrm-social/bookwyrm#2826] [Ideas]
I'm an author and regular user of Bookwyrm. I came here looking for issues regarding author functions, and I wanted to start a discussion about a roadmap for author users.

Features that I could see as super useful to start:

1) Authors connect their books to their bookwyrm accounts (see https://github.com/bookwyrm-social/bookwyrm/issues/2822). 
2) Editable author profiles
3) Author dashboard, with quick access to their books, reviews of their books, adding or editing book details, stats, etc.

I do think adding these features would be a boost for bookwyrm adoption.

I did do some Django development for a while, and although I am retired from coding, I could collaborate with someone interested in this who is steeped in the codebase - I don't really have the bandwidth to get up to speed with it. (I could collaborate on model and  template ideas, etc.)

**Comment by jaschaurbach** [comment]
This would work without any problem if there would be just one instance running bookwyrm. 

I do not See that an instance is becoming the truth regarding "this is the author" to all instances _or_ an author needs to create an account on all instances where they want to have these features.

**Comment by maxpearl** [comment]
Yes, each instance has it's own user accounts, so I get that - does each instance also have it's own library of books? I'm pretty new to the fediverse in terms of the underlying innards.

So in terms of # 2 and 3 above, bookwyrm could have those features, and all you would need is a way to link a book's author with their canonical instance. They would have their profile and account on only one instance, but couldn't the links to the author (just like links for users in the fediverse) could point back to the profile on the instance the author is using?

Part of the challenge is that "claims" for authoring books would have to somehow be verified - that sounds complicated. Although that could be a moderator task, the interface offers a place for users to claim books and add evidence, and then a moderator does the linking?

**Reply by jaschaurbach** [reply]
Every instance can create editions of books. This is the whole federations thing. 

And Ohne an instance never communicated with an instance where an author is on this information never gets to the instance. What is with instances which block each others for $reasons? The whole idea would work with a central book database which we do not have (luckily) which is the whole point of decentralization.

**Reply by maxpearl** [reply]
OK, I get the decentralization thing, and clearly each instance has it's own library of books. There still has to be a way, given this architecture, for an author to claim their books. Even just to start giving the ability for an author to claim their books on one instance is useful, and couldn't author claims be federated? It seems that leaving authors completely out of the design of this is kinda shortsighted. I get that the decentralized architecture makes things a little more challenging to architect, but I can't believe this isn't possible.

**Reply by mouse-reeve** [reply]
This has been on my mind for ages and hasn't happened yet. There's some initial thoughts on it here: https://github.com/bookwyrm-social/bookwyrm/issues/1311
